.drone/drone.yml

```diff
@@ -23,7 +23,16 @@
          ],
          "image": "golang:1.16.7",
          "name": "make-test"
-      }
+      },
+     {
+       "commands": [
+          "apt-get update && apt-get -y install protobuf-compiler",
```

Comment:
It's not clear what version of the compiler and protoc-gen-gogoslick is being installed here. This seems like it would create issues when the version installed here doesn't match what a developer compiled with locally. Should dskit have its own build image ala Cortex?

Reply:
Yeah I was trying to figure out if it was worth a build image for this as well. I'd prefer not to make a build image if we can avoid it. I think you can specify a version for at least protoc-gen-gogoslick.

Reply:
We could also download the protobuf-c binary to ./tools and use that rather than apt-get here

Reply:
> We could also download the protoc binary to ./tools and use that rather than apt-get here

That seems reasonable to me since it would ensure that people were running the same version locally as CI but I'd be curious to hear what other people think

Reply:
I took a quick run at trying that method, it seems to work with drone and locally. We can also set versions that way.

Reply:
I tried to build this to match what we use in cortex today to prevent a potential versioning issue with our proto-files. Is there a way to make sure these versions are what get vendored for local builds? We seem to rely on our build image for cortex for maintaining the versions.